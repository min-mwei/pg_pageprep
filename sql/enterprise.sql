/* minimal delays */
SET pg_pageprep.per_relation_delay=0;
SET pg_pageprep.per_page_delay=0;

CREATE FUNCTION show_reloptions(rel regclass) RETURNS text[] AS
$$
	SELECT reloptions FROM pg_class WHERE oid = rel;
$$ LANGUAGE sql;

create extension pg_pageprep;
create table test1(a int) with (fillfactor=80);

-- insert fake value
insert into pg_pageprep_jobs values (
	'test1'::regclass, 50, 'done', 1
);

select show_reloptions('test1'::regclass);
select restore_fillfactors();
select show_reloptions('test1'::regclass);
