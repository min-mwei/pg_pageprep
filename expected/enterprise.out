/* minimal delays */
SET pg_pageprep.per_relation_delay=0;
SET pg_pageprep.per_page_delay=0;
CREATE FUNCTION show_reloptions(rel regclass) RETURNS text[] AS
$$
	SELECT reloptions FROM pg_class WHERE oid = rel;
$$ LANGUAGE sql;
create extension pg_pageprep;
create table test1(a int) with (fillfactor=80);
-- insert fake value
insert into pg_pageprep_jobs values (
	'test1'::regclass, 50, 'done', 1
);
select show_reloptions('test1'::regclass);
 show_reloptions 
-----------------
 {fillfactor=80}
(1 row)

select restore_fillfactors();
 restore_fillfactors 
---------------------
 
(1 row)

select show_reloptions('test1'::regclass);
 show_reloptions 
-----------------
 {fillfactor=50}
(1 row)

